test/docker-compose.integration-tests.yml

```diff
@@ -0,0 +1,105 @@
+version: "3.8"
```

Comment:
it looks like we may have lost the e2e tests in this migration. whatever, i can salvage

Reply:
@wimax-grapl this shouldn't remove the e2e tests, they should still exist in dobi and such. Even my branch, my changes don't currently impact the current system.

Reply:
We should do everything we can to not depend on two build systems at once. Let's get rid of dobi altogether if that's what we're doing. There's no point to merging code for unused build system components because it'll go stale rapidly.